dashboard/apps/dashboard/management/commands/sync.py

```diff
@@ -111,6 +112,28 @@ def sync_clients(data_dir):
             client.save()
 
 
+def sync_departments(data_dir):
+    logging.info('sync departments')
+    source = os.path.join(data_dir, 'departments.json')
+    with open(source, 'r') as sf:
+        data = json.loads(sf.read())
+    for item in data:
+        useful_data = {
+            'float_id': item['department_id'],
+            'name': item['department_name'],
+            'raw_data': item,
+        }
```

Comment:
Not sure if you are aware of the context.

This module is very specific to float.com the external scheduling system. It's better to keep this as the single source of the mapping from Float's data model to our data model. How the attributes are named in Float, e.g. department_id, department_name etc, is not what we need to keep as a contract. Imagine if we need to sync from another data source, the attributes will be named differently. Or imagine if Float decides to change the names (they've done it before), we don't need to worry about changing our model and breaking the contract to code using the `from_json` method.